fighthealthinsurance/static/css/main.css

```diff
@@ -506,7 +506,7 @@ header span i {
 }
 
 .slider .item-first {
-    background-image: url(../images/letter.jpeg);
+    background-image: url(../images/crinkledcolors.jpg);

```

Comment:
eh I'm not sold on this change. I

Reply:
haha i know! i just wanted to raise it and leave it here so you can look at the pretty colorful crinkled paper